tests/test_filterDiaSourceCatalog.py

```diff
@@ -241,13 +303,21 @@ def test_run_with_all_filters(self):
         filterDiaSourceCatalogTask = FilterDiaSourceCatalogTask(config=self.config)
```

Comment:
Not very important, but note that the docstring for this test should be updated (there are not 6 sources)

Reply:
Thanks for spotting this! I also noticed I also forgot to change the docstring for one of the tests I added, so I'll fix that as well.